comment.go

```diff
@@ -74,3 +75,20 @@ func getComments(url string) ([]Comment, error) {
 
 	return comments, nil
 }
+
+func countComments(url string) (int, error) {
+	var count int
+
+	statement := `
+		SELECT COUNT(*) FROM comments WHERE url=?
+	`
+	err := db.QueryRow(statement, url).Scan(&count)
+	switch {
+	case err == sql.ErrNoRows:
```

Comment:
I didn't know Go allowed cases like this in `switch` :3 

I think a `if(condition) { ... } else if(condition) { ... } else { ... }` would be simpler here